pkg/stacks/constants.go

```diff
@@ -10,9 +10,15 @@ const (
 type PayloadType byte
 
 const (
-	PayloadTypeTokenTransfer PayloadType = 0x00
-	PayloadTypeSmartContract PayloadType = 0x01
-	PayloadTypeContractCall  PayloadType = 0x02
+	PayloadTypeTokenTransfer          PayloadType = 0x00
+	PayloadTypeSmartContract          PayloadType = 0x01
+	PayloadTypeContractCall           PayloadType = 0x02
+	PayloadTypePoisonMicroblock       PayloadType = 0x03
+	PayloadTypeCoinbase               PayloadType = 0x04
+	PayloadTypeCoinbaseToAltRecipient PayloadType = 0x05
+	PayloadTypeVersionedSmartContract PayloadType = 0x06
+	PayloadTypeTenureChange           PayloadType = 0x07
+	PayloadTypeNakamotoCoinbase       PayloadType = 0x08
```

Comment:
I did not include payload types outside of transfer, contract call, and contract deploy because I did not need to implement them for my purposes. I think that they should only be added if they are implemented. What do you think?

Reply:
The SDK is not only for initiating transactions, but also for parsing and analyzing. I added all, but only VersionedSmartContract is implemented. Post-nakamoto we'll need to deploy ClarityVersion3 contracts.